src/main/resources/avro/bdg.avdl

```diff
@@ -404,6 +404,23 @@ record Variant {
   union { null, string } alternateAllele = null;
 
   /**
+   True if filters were applied for this variant. VCF column 7 "FILTER" any value other
+   than the missing value.
+   */
+  union { null, boolean } filtersApplied = null;
```

Comment:
This SGTM, but is inconsistent with the filter fields in the `Genotype` record (unless we already renamed them, hard to see in the file browser). I think the structure you're using here is slightly better; the `Genotype` record rolls `filtersApplied` and `filtersPassed` into a single field and encodes "filters not applied" by nulling the field. I'd just like the two to be consistent is all.